templates/offer_type/_form.html.twig

```diff
@@ -0,0 +1,57 @@
+<section class="margin">
+    {{ form_start(form, {'attr': {'novalidate': 'novalidate'}}) }}
+    <div id="new-offer-form" class="p-4">
+        <h3 class="mt-4">Job description</h3>
+        <div class="col mt-4">
+            {{ form_row(form.title) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.startAt) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.contract) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.workFromHome) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.description) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.experience) }}
+        </div>
+        <div id="upload-offer-picture-form" class="col mt-4">
+            {{ form_row(form.offerPicture) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.minSalary) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.maxSalary) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.location) }}
+        </div>
+        <div class="col mt-4">
+            {{ form_row(form.interviewProcess) }}
+        </div>
+        <div class="col mt-4">
+        <h3 class="mt-2 mb-4"> Required Hard skills</h3>
+        {{ form_errors(form.skills) }}
+            {% for label, value in field_choices(form.skills) %}
```

Comment:
Est-ce que tu pourrais indenter le for avec le h3 pour qu'ils soient bien alignés? Merciii